Proofpoint observe it (#25427)
demisto/content: Packs/ProofpointObserveIT/README.md

# Proofpoint ObserveIT
This pack includes Cortex XSIAM content. 

## Configuration on Server Side
You need to configure Proofpoint ObserveIT CEF log integration.

Follow the below steps to configure CEF log integration:

> 1. Go to **Configuration** &rarr;  **Integrations** &rarr; **Integrated SIEM** 

Reviewer: ```suggestion
1. Go to **Configuration** > **Integrations** > **Integrated SIEM**.
```

> 2. Click on the **SIEM Log Integration** tab.

Reviewer: ```suggestion
2. Click the **SIEM Log Integration** tab.
```

> 3. Select the **Enable export to ARCSight format** check box.

Reviewer: ```suggestion
3. Select the **Enable export to ARCSight format** checkbox.
```

> 4. In the **Log data** section select **Activity Alerts** (should be selected by default).

Reviewer: ```suggestion
4. In the **Log data** section, select **Activity Alerts** (should be selected by default).
```

5. In the **Log file properties** section, accept the default log file location and name:
   - Path: C:\Program File\ObserveIT\NotificationService\LogFiles\ArcSight
   - Name: Observeit_activity_log.cef

> 6. In the **Log file cleanup** section schedule the frequency for clearing the log file.

Reviewer: ```suggestion
6. In the **Log file cleanup** section, schedule the frequency for clearing the log file.
```

7. Click **Save**.

> More information can be found [here](https://documentation.observeit.com/configuration_guide/configuring_cef_log_integratio)

Reviewer: ```suggestion
More information can be found [here](https://documentation.observeit.com/configuration_guide/configuring_cef_log_integration.htm)
```
